sanic/exceptions.py

```diff
@@ -38,10 +38,10 @@ def __init__(self, message, status_code=None, quiet=None):
             self.quiet = quiet
         elif hasattr(type(self), "quiet"):
             pass
-        elif status_code not in (None, 500):
-            self.quiet = True
-        else
+        elif status_code in (None, 500):
```

Comment:
Why this change?

I'd rather see it without the multiple conditionals.

```python
else:
    self.quiet = status_code not in (None, 500)
```

Reply:
OK.